create-record/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "create-record",
+  "version": "1.0.2",
+  "description": "A serivce that takes in a JSON payload and puts it in an encrypted s3 bucket.",
+  "main": "index.js",
+  "dependencies": {
+    "aws-sdk": "^2.649.0",
+    "aws-sdk-mock": "^5.1.0",
+    "nodeunit": "^0.11.3",
```

Comment:
As these are purely testing depedencies they should be added as devDependencies. 

https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file

This can be done from the command line by running `npm i --save-Dev aws-sdk-mock` or by manually copy into a devDependcies section in package.json